### ISSUE TITLE
Windows desctop app doesn't work

### ISSUE THREAD

**Issue by SurmatMG** [mike-goodwin/owasp-threat-dragon-desktop#152]
Hello, guys! I am trying to create new or open saved locally at json threat models, but it doesn't work. I am using Windows 10, and perform all the instruction from the site. Could you help me?

![image](https://user-images.githubusercontent.com/69574697/89989064-10e08000-dc89-11ea-9142-18277123a977.png)

**Comment by jgadsden** [comment]
Hello @SurmatMG - are you using the latest release version 1.2? you can download it here:
https://github.com/mike-goodwin/owasp-threat-dragon-desktop/releases/tag/v1.2

and then the best way to get started is to open a sample model. This is a good option if you are new to Threat Dragon.

**Comment by SurmatMG** [comment]
Yes, I am using 1.2, and I am trying to open sample model, but saving
doesn't work!

So any changes to basic model even if I saved them are not opened!

What should I do?

**Comment by jgadsden** [comment]
Thanks for the update @SurmatMG - so you can open a sample model, but when you try and save you are not able?

Can you give details on where you are trying to save to model to? and whether this directory has write permissions for you?

Thanks, Jon

**Comment by jgadsden** [comment]
@SurmatMG - are you getting an exception thrown ?

There is an open issue with TD desktop on Windows 10 that could not be reproduced:
https://github.com/OWASP/threat-dragon-desktop/issues/3

**Comment by SurmatMG** [comment]
"Thanks for the update @SurmatMG <https://github.com/SurmatMG> - so you can
open a sample model, but when you try and save you are not able?"

1. I am able to open, but I can not save my edition. (saved file doesn't
open).

"Can you give details on where you are trying to save to model to? "

2. I tried different directories and I have root access to them (so
everything is ok with write permissions), files saved but Thread Dragon
doesn't open them (see a. 1 above).

**Comment by SurmatMG** [comment]
" are you getting an exception thrown ?"

No, there is nothing - no exceptions, no errors.

**Comment by jgadsden** [comment]
Hello @SurmatMG , I must admit that I am not a windows sort of person - I contribute to Threat Dragon using Linux and MacOS - and only test on a Windows 10 machine for the releases. It sounds like the dialog boxes are not being displayed in your Windows environment, it works on the test machines but these may be set up differently.

@mike-goodwin and @andk123 - I think you develop on windows, do you have any ideas on this?

**Comment by dschadt** [comment]
some antivirus tools block access to protected folders, I see that e.g. with Bitdefender.

**Comment by andk123** [comment]
Unfortunately, I am not able to reproduce the problem. Some of the issues I had in the past with similar releases were compatibility issues when installing Threat Dragon which were solved by using Right-click then "Troubleshoot compatibility" on Windows. Otherwise if the program is launching fine, and you are able to open the sample model, then I am not sure what is causing that. @dschadt brought up an interesting point. You could also maybe try the "Run as administrator" option to launch the program. Let me know if you get any result.

**Comment by dewil-official** [comment]
Just had the same issue. Downgrading to 1.1 didn't work. Fresh installation on Windows 10 (latest build)

I managed to fix it by accident by not storing the file on my NAS, but on a local drive. Seems like the tool cannot handle local network paths? At least not for opening the diagram-editor.

**Comment by jgadsden** [comment]
Hello @dewil-official - thanks for investigating why it was not working. I must admit that I do not test for network drives, so had not seen this issue. We are planning version 2.0 of Threat Dragon for later this year so we can make sure we test  for this on the new version.

Thanks again, Jon
